README.md

```diff
@@ -10,10 +10,34 @@ ALCS are is an extension to basic LCS compromising the notation of anticipations
 
 
 ## Deep dive
+
+### Project setup
+
+First, create a [virtualenv](https://virtualenv.pypa.io/) for the project:
```

Comment:
It's ok. But please note that it is one of many possibilities (virtualenv/virtualenvwrapper/conda environments).

Reply:
Sure, I'll modify the instructions a bit